src/sentry/integrations/github/client.py

```diff
@@ -213,6 +213,48 @@ def get_file(self, repo: Repository, path: str, ref: str) -> str:
         encoded_content = contents["content"]
         return b64decode(encoded_content).decode("utf-8")
 
+    def get_blame_for_file(
+        self, repo: Repository, path: str, ref: str
+    ) -> Sequence[Mapping[str, Any]]:
```

Comment:
Would you mind adding a test for this code?
Do you have a sample URL to see this in the GraphQL explorer?